content/pt-br/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,158 @@
+- Se você mantém seu próprio registry de imagens, voc6e pode copiar as imagens que você precisa para reduzir o tráfego
+ao registry da comunidade.
+
+Se você acha que vai ser impactado, ou gostaria de saber mais sobre essa mudança, continue a leitura.
+
+## Como eu posso checar se eu vou ser afetado?
+Para testar a conectividade ao registry.k8s.io (e saber se consegue fazer o pull de imagens), pode executar o
+comando abaixo em um namespace da sua escolha:
+
+```
+kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
+```
+
+Quando executar o comando acima, é esperada a seguinte saída caso tudo esteja funcionando corretamente:
+
+```
+$ kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
+Fri Feb 31 07:07:07 UTC 2023
+pod "hello-world" deleted
+```
+
+## Quais erros são esperados caso eu seja afetado?
+Os erros podem depender do tipo de container runtime que está usando, e de qual endpoint você está 
+roteando para, mas devem ser erros como `ErrImagePull`, `ImagePullBackOff`, ou falha na criação do
+container com o aviso `FailedCreatePodSandBox`.
+
+Abaixo um exemplo de uma mensagem de erro mostrando um deployment de proxy falhando devido um certificado desconhecido:
+
+```
+FailedCreatePodSandBox: Failed to create pod sandbox: rpc error: code = Unknown desc = Error response from daemon: Head “https://us-west1-docker.pkg.dev/v2/k8s-artifacts-prod/images/pause/manifests/3.8”: x509: certificate signed by unknown authority
+```
+
+## Quais imagens serão afetadas?
+**TODAS** as imagens no k8s.gcr.io serão afetadas por essa mudança. O k8s.gcr.io hospeda muitas imagens além das releases do Kubernetes. Um grande número de subprojetos do Kubernetes hospedam seus projetos nele também. Alguns exemplos incluem 
+as imagens `dns/k8s-dns-node-cache`, `ingress-nginx/controller`, e
+`node-problem-detector/node-problem-detector`.
+
+## Eu fui afetado, o que devo fazer?
+Para usuários afetados que usam um ambiente restrito, a melhor opção é copiar as imagens necessárias
+para um registro privado ou configurar um cache de pull-through nos seu registry.
+
+Existem várias ferramentas para copiar imagens entre registries. [Crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_copy.md) é uma dessas ferramentas, as imagens podem ser copiadas para um registry privado com `crane copy SRC DST`. Também existem ferramentas específicas de fornecedores, como o [gcrane](https://cloud.google.com/container-registry/docs/migrate-external-containers#copy) que faz uma função similar mas simplificada para a plataform da Google.
+
+
+## Como eu posso encontrar quais imagens estão usando o registry antigo, e corrigir elas?
+
+**Opção 1**: Veja esse comando do kubectl no [blog post anterior](https://kubernetes.io/blog/2023/02/06/k8s-gcr-io-freeze-announcement/#what-s-nex):
```

Comment:
There is a pt-br link “ /pt-br/blog/2023/02/06/k8s-gcr-io-freeze-announcement/#próximos-passos”